packages/edit-site/src/components/page/header.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	__experimentalHeading as Heading,
+	__experimentalText as Text,
+	__experimentalHStack as HStack,
+	FlexBlock,
+	FlexItem,
+} from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+
+export default function Header( { title, subTitle, actions } ) {
+	return (
+		<HStack as="header" alignment="left" className="edit-site-page-header">
+			<FlexBlock className="edit-site-page-header__page-title">
```

Comment:
Nit: We can use [`<hgroup>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/hgroup) to group the heading with its subheading here. It doesn't seem to have any accessibility impact though so it's just a fun TIL moment and don't need to change 😅 .